buckw

```diff
@@ -213,10 +213,11 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    # Go directly to the kill command, help command, or buck options. Do not run okbuck.
+   skip=("kill" "help" "--help" "-h" "--version" "-V")
```

Comment:
I would not put `kill` and `help` here.

because those are valid target names. we should only skip if kill is `$1`